caps.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const faker = require('faker');
+
+const Events = require('events');
+
+const events = new Events();
+
```

Comment:
instead of `const Events = require('events');` and ` const events = new Events();` you can replace this part with `const events = require('./events.js')`.  You will also do the same thing for the other two files (vendor and driver).